www/html/ci/application/views/navbar.html

```diff
@@ -7,17 +7,17 @@
 {% block head %}{% endblock %}
 </head>
 <body>
-<div class="container kt-tabs-component">

```

Comment:
Why did you remove the class here? Aren't we losing all the other css this div used to have?